package.json

```diff
@@ -13,7 +13,8 @@
     "prepublish": "npm run build",
     "prepush": "npm test",
     "pretest": "npm run lint",
-    "test": "NODE_ENV=test nyc mocha"
+    "test": "NODE_ENV=test nyc mocha",
+    "tdd": "NODE_ENV=test nyc mocha --watch"
```

Comment:
trivial, i also like to add `--break` in TDD as well to focus on the first error

Reply:
updated